database/src/main/kotlin/no/nav/su/se/bakover/database/skatt/SkattPostgresRepo.kt

```diff
@@ -61,10 +70,12 @@ internal object SkattPostgresRepo {
         session: Session,
     ) {
         """
-                    insert into
-                        skatt (id, sakId, fnr, erEps, opprettet, saksbehandler, årSpurtFor, data)
-                    values
-                        (:id, :sakId, :fnr, :erEps, :opprettet, :saksbehandler, to_json(:aarSpurtFor::jsonb), to_json(:data::jsonb))
+            insert into
+                skatt (id, sakId, fnr, erEps, opprettet, saksbehandler, årSpurtFor, data)
+            values
+                (:id, :sakId, :fnr, :erEps, :opprettet, :saksbehandler, to_json(:aarSpurtFor::jsonb), to_json(:data::jsonb))
+            on conflict (id) do update set
+                opprettet=:opprettet, saksbehandler=:saksbehandler, årSpurtFor=to_json(:aarSpurtFor::jsonb), data=to_json(:data::jsonb)
```

Comment:
Må vel oppdatere fnr og erEps?

Reply:
Dette er egentlig bare for endring av stønadsperiode. Hvis man fjerner endrer eps sin Fnr, så blir den jo slettet, og man må søke opp ny